src/Nebulas/Address.h

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../PublicKey.h"
+
+#include <array>
+#include <cstdint>
+#include <vector>
+
+namespace TW::Nebulas {
+
+class Address {
```

Comment:
we have a class called `Base58Address`, please reuse it if possible

Reply:
Ok, Nebulas::Address is inherited from Base58Address now.